xtend-website/_posts/releasenotes/2024-04-22-version-2-35.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  Xtend 2.35.0 Release Notes
+date:   2024-04-22
```

Comment:
ususally we pick the tuesday of the rc1 week as file name and release date

Reply:
@cdietrich where do I find such information?

Reply:
https://github.com/eclipse-simrel/.github/tree/main/wiki/SimRel